transport.go

```diff
@@ -257,8 +257,8 @@ func (t *transport) holePunch(network string, addr *net.UDPAddr) error {
 		return err
 	}
 
-	for i := 0; i < 5; i++ {
-		time.Sleep(time.Duration(rand.Intn(i * int(time.Millisecond))))
+	for i := 0; i < 10; i++ {
+		time.Sleep(time.Duration(rand.Intn(i * int(10*time.Millisecond))))
```

Comment:
@vyzo We already have a backoff-retry logic in the Hole Punching protocol. Do we need this ?

Reply:
Can you explain why we need this ?

Reply:
Yes, I think so; we should send multiple packets and try to coincide (within a few ms) with the inbound connection packet.